app/customer.py

```diff
@@ -0,0 +1,36 @@
+from app.shop import Shop
+
+
+class Customer:
+    def __init__(
+            self,
+            name: str,
+            products_cart: dict,
+            location: list[int],
+            money: int | float,
+            car: dict
+    ) -> None:
+        self.name = name
+        self.products_cart = products_cart
+        self.location = location
+        self.money = money
+        self.car = car
+
+    def trip_price(self, fuel_price: int | float, shop: Shop) -> int | float:
```

Comment:
Try to separate this function into a few functions with different responsibilities. like get_distance, get_fuel_cost and others